src/index.js

```diff
@@ -63,12 +63,24 @@ function Debug(options) {
   };
 
   common.Service.call(this, config, options);
+
+  // FIXME(ofrobots): We need our own copy of options because Service may
+  // default to '{{projectId}}' when options doesn't contain the `projectId`.
+  // property. This breaks the SSOT principle. Remove this when
+  // https://github.com/GoogleCloudPlatform/google-cloud-node/issues/1891
+  // is resolved.
+  this.options = options;
 }
 util.inherits(Debug, common.Service);
 
 var initConfig = function(config_) {
   var config = config_ || {};
 
+  if (config.keyFilename || config.credentials || config.projectId) {
+    throw new Error('keyFilename, projectId or credentials should be provided' + 
+                    ' in the module options');
```

Comment:
Do we use the term "module" options elsewhere in our documentation? This could be confusing if people aren't clear on the different options objects we deal with.